pkgs/development/python-modules/tensorflow/default.nix

```diff
@@ -103,27 +109,21 @@ let
       owner = "tensorflow";
       repo = "tensorflow";
       rev = "v${version}";
-      sha256 = "sha256-ncwIkqLDqrB33pB9/FTlBklsIJUEvnDUmyAeUfufCFs=";
+      sha256 = "0yl06aypfxrcs35828xf04mkidz1x0j89v0q5h4d2xps1cb5rv3f";
     };
 
     patches = [
-      # Fixes for NixOS jsoncpp
-      ./system-jsoncpp.patch
```

Comment:
Does it still uses the system jsoncpp? If not please remove it from buildInputs.

Reply:
Please also remove the patch file

Reply:
It still should use system jsoncpp but it doesn't need the patch anymore. Removed the patch.